frontend/README.md

```diff
@@ -0,0 +1,13 @@
+## SOLO frontend
+
+This react app is the frontend to the System for Operational Logistics Orders (SOLO)
+
+Scripts:
+  - ```npm install``` Install dependencies: 
+  - ```npm run start``` Run local development server
+  - ```npm run test``` Run tests
+  - ```npm run lint``` Lint
+  - ```npm run lint:fix``` Autofix linting
+  - ```npm run typecheck``` Type-check
+  - ```npm run build``` Create production build
+  - ```docker build .``` Build production app into nginx container (requires docker installed)
```

Comment:
We should add steps to get dev env started. Should be simple. Or do we have another place we will keep that info?